data/transactions/logic/backwardCompat_test.go

```diff
@@ -399,37 +396,24 @@ func TestBackwardCompatGlobalFields(t *testing.T) {
 
 		ops := testProg(t, text, AssemblerMaxVersion)
 
-		proto := config.Consensus[protocol.ConsensusV23]
```

Comment:
why did it go? I'd like to do not modify these backward compat tests if possible

Reply:
Because the goal was achieved by giving a version argument to `defaultEvalParamsWithVersion`